radix_sort.py

```diff
@@ -0,0 +1,63 @@
+"""This module contains the radix sort method (radix_srt), which performs an
+in-place sort on a passed in list. Radix sort has a best case time
+complexity of O(n log n). This implementation of radix sort is stable, but not
+adaptive.
+
+References:
+
+Radix Sort in Python
+http://www.geekviewpoint.com/python/sorting/radixsort
+by Isai Damier
+"""
+
+
+def radix_srt(un_list):
+    """Sort a list in place with radix sort.
+
+    args:
+        un_list: the list of positive integers to be sorted.
+    """
+    radix = 10
+    maxLen = False
+    tmp, placement = -1, 1
+
+    while not maxLen:
+        maxLen = True
+        buckets = [list() for num in range(radix)]
+
+        for i in un_list:
+            tmp = i // placement
+            buckets[tmp % radix].append(i)
```

Comment:
given that floored division and the `%` operator are quite expensive, can you think of a less CPU intensive way of calculating the needed values here?